app/src/main/java/app/gamenative/service/epic/EpicCloudSavesManager.kt

```
@@ -0,0 +1,1340 @@
+            SyncAction.NONE
+        }
+    }
+
+    /**
+     * Parse Epic timestamp string to milliseconds
+     */
+    private fun parseTimestamp(timestamp: String): Long {
+        return try {
+            val instant = Instant.parse(timestamp)
+            instant.toEpochMilli()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)